app/imports/ui/pages/CreateTestimony.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import '/client/style.css';
+import { Col, Container, Image, Row } from 'react-bootstrap';
+import { PAGE_IDS } from '../utilities/PageIDs';
+
+const CreateTestimony = () => (
+  <Container id={PAGE_IDS.CREATETESTIMONY} className="py-3">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous" />
```

Comment:
I dont think you have ESLint installed with the right codestyle, we can try fix in class but basically from line 8 to line 88 it should be indented by 2 more white spaces.

Reply:
@lisemnilsen 
Format code on save.
![Screenshot from 2022-09-08 11-32-24](https://user-images.githubusercontent.com/26912258/189809758-bd95093d-4433-4c11-a9a3-a130ef39a558.png)

Reply:
Thanks, but I use visual studio code.. but I think I fixed it:)